packages/integration-tests/src/helper.ts

```diff
@@ -0,0 +1,89 @@
+import { execSync } from "child_process";
+import { ethers } from "ethers";
+import { promises as fs } from "fs";
+import * as path from "path";
+import * as request from "supertest";
+import { Provider } from "zksync-web3";
+
+import { environment, localConfig } from "./config";
+import { Logger } from "./entities";
+
+import type { BaseProvider } from "@ethersproject/providers/src.ts/base-provider";
+
+export class Helper {
+  async txHashLogger(txType: string, txValue: string, tokenName?: string) {
+    const logMessage = `TxHash for ${txType} ${Logger.textSeparator} ${txValue}`;
+
+    if (tokenName === undefined) {
+      return console.log(logMessage);
+    } else {
+      return console.log(logMessage, ` ${tokenName}`);
+    }
+  }
+
+  async executeScript(script: string) {
+    const output = execSync(script, { encoding: "utf-8" });
+
+    try {
+      console.log(`> Run NPM Script "${script}":\n`, output);
+      return output;
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  async getStringFromFile(fileName: string) {
```

Comment:
The name is confusing, it should just be `readFile`. I think it's `filePath` not `fileName` in the args. What's the point of adding this `..`? You can just include it in the place where you call it. Or otherwise just pass a filename without path and call it like `readFileFrom{FOLDER_NAME}`.

Reply:
updated, changed all variables and replaced the path in constants.ts